File: frontend/fawryfrontpart/my-angular-app/src/app/MovieOMDBAPI.ts
export class MovieOMDBAPI {
    Title: string;
    Year: string;
    Genre: string;
    Director: string;
    Plot: string;
    Poster: string;
  
    constructor(
      Title: string,
      Year: string,
      Genre: string,
      Director: string,
      Plot: string,
      Poster: string
    ) {
      this.Title = Title;
      this.Year = Year;
      this.Genre = Genre;
      this.Director = Director;
      this.Plot = Plot;
      this.Poster = Poster;
    }
  }
  

File: backend/fawryproject/MovieApplication/MovieApplication/src/main/java/com/Fawry/MovieApplication/dto/Movie/MovieOMDBAPI.java
package com.Fawry.MovieApplication.dto.Movie;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Data
public class MovieOMDBAPI {
    @JsonProperty("Title")
    private String Title;
    @JsonProperty("Year")
    private String Year;
    @JsonProperty("Genre")
    private String Genre;
    @JsonProperty("Director")
    private String Director;
    @JsonProperty("Plot")
    private String Plot;
    @JsonProperty("Poster")
    private String Poster;

}
